README.md

```diff
@@ -5,7 +5,9 @@ Dockstore Load/Performance/Stress Tester
 
 ### Requirements
 
-You need Java 11 installed.
+Running the tests works with Java 17. It's not working with Java 21, I haven't looked into why.
+
+Running the comparison tool, gatling-report-6.0-capsule-fat.jar, seems to require Java 8 (maybe it works with 11, I don't have it installed).
```

Comment:
If this uses capsule, I remember an issue with capsule and nextflow using Java 17 
https://github.com/nextflow-io/nextflow/issues/2856